pulpcore/pulpcore/app/serializers/task.py

```diff
@@ -101,6 +101,14 @@ class Meta:
                                                 'created_resources')
 
```

Comment:
The DRF way to do this (if I understand it correctly) is to specify the list serializer in the TaskSerializer Meta.

    list_serializer_class = MinimalTaskSerializer(serializers.ListSerializer)

http://www.django-rest-framework.org/api-guide/serializers/#listserializer